test/pylib/rest_client.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""Asynchronous helper for Scylla REST API operations.
+"""
+import os.path
+import aiohttp
+
+
+class UnixRESTClient():
+    """An async helper for REST API operations using AF_UNIX socket"""
+    session: aiohttp.ClientSession
+
+    def __init__(self, sock_path: str):
+        self.sock_name = os.path.basename(sock_path)
+        self.session = aiohttp.ClientSession(connector = aiohttp.UnixConnector(path=sock_path))
+
+    async def stop(self) -> None:
```

Comment:
This function is not used anywhere.

Reply:
Function now used in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d